tests/test_isPrimaryFlag.py

```diff
@@ -165,9 +165,6 @@ def setUp(self):
         charImTask = CharacterizeImageTask(config=charImConfig)
         self.charImResults = charImTask.run(self.exposure)
 
-        # set log level so that warnings do not display
-        logging.getLogger("lsst.calibrate").setLevel(logging.ERROR)
-
```

Comment:
Does this make some warnings appear?

Reply:
Depends what you mean. They don't appear by default anywhere. Pytest captures warnings and reports them if the tests failed.